src/settings/lib/RuleEditor/RulesEditor.js

```diff
@@ -424,41 +387,34 @@ class RulesEditor extends React.Component {
       customKeys:{
         'Up': moveFocusUp,
         'Down': moveFocusDown,
-        'PageUp': function(cm, handle) { handle.moveFocus(-handle.menuSize() + 1, true); },
-        'PageDown': function(cm, handle) { handle.moveFocus(handle.menuSize() - 1, true); },
-        'Home': function(cm, handle) { handle.setFocus(0); },
-        'End': function(cm, handle) { handle.setFocus(handle.length - 1); },
+        'PageUp': (codeMirror, handle) => { handle.moveFocus(-handle.menuSize() + 1, true); },
+        'PageDown': (codeMirror, handle) => { handle.moveFocus(handle.menuSize() - 1, true); },
+        'Home': (codeMirror, handle) => { handle.setFocus(0); },
+        'End': (codeMirror, handle) => { handle.setFocus(handle.length - 1); },
         'Enter': handleEnter,
         'Tab': handleTab,
-        'Esc': function(cm, handle) { handle.close },
+        'Esc': noop,
```

Comment:
Could you please clarify why the function was replaced by noop from lodash?

Reply:
`handle.close` is not called at all, so basically the whole function is not doing anything.